routing/index_graph.cpp

```diff
@@ -48,6 +48,13 @@ void IndexGraph::GetNeighboringEdges(Segment const & from, RoadPoint const & rp,
                                      vector<SegmentEdge> & edges)
 {
   RoadGeometry const & road = m_geometry.GetRoad(rp.GetFeatureId());
+
+  // Note. It's possible that it's used a map which is built with car_model different from
+  // car_model in current sources. For example, the map from 12.2016 contained highway==pedestrian
+  // in car_model but now this type of highways is removed. In such cases |road| has no points.
```

Comment:
// Note. It's possible that car_model was changed after the map was built.
// For example, the map from 12.2016 contained highway=pedestrian
// in car_model but this type of highways is removed as of 01.2017.
// In such cases |road| has no points.